perf-event/src/lib.rs

```diff
@@ -432,6 +451,137 @@ pub struct CountAndTime {
     pub time_running: u64,
 }
 
+bitflags::bitflags! {
+    /// Specifies which fields to include in the sample.
+    ///
+    /// These fields will be recorded in the [`Sampler`] output buffer.
+    ///
+    /// [`Sampler`]: crate::Sampler
+    #[derive(Default)]
+    pub struct Sample : u64 {
```

Comment:
For the sake of API, it makes sense to have `Sampler` and `Sample` available in the crate's top-level module - but for the sake of code organization, it feels like this belongs in `samplers/mod.rs`.

Reply:
Yeah, that makes sense. I've moved `Sample` to the `samples` module and renamed it `SampleType` so that it doesn't conflict with a future `RECORD_TYPE_SAMPLE` binding.